egs/apiai_decode/s5/download-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
+
+DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
+
```

Comment:
Interesting idea is to add alternative models into this script. Like Librispeech. Maybe having some models repo of some sort, if kaldi will be getting more popular. Just random thought.